src/publisher/middleware.py

```diff
@@ -143,15 +57,25 @@ def requested_version(request, response):
     return (response_mime[0], "*" if not versions else max(versions))
 
 
-def deprecated(request):
-    if not settings.API_V12_TRANSFORMS:
-        return False
-    targets = [
-        ("application/vnd.elife.article-poa+json", "version", b"1"),
-        ("application/vnd.elife.article-vor+json", "version", b"1"),
-    ]
+# deprecated content-types are simply the oldest ones
+
+DEPRECATED_VOR = (
+    "application/vnd.elife.article-vor+json",
+    "version",
+    bytes(str(settings.ALL_SCHEMA_IDX["vor"][-1][0]), "utf-8"),
```

Comment:
A smart way to deprecate the oldest with `-1` 👍

Reply:
yeah ... I found I was hardcoding versions all over the place in the code. Back when it was just a change from v1 to v2 it wasn't such a big deal and the bump from v2 to v3 was trivial but now it's getting tiresome ;)

I wonder if the RAML tooling for python has changed much since when we first adopted it. It was brand new back then without much adoption.